webextensions/chrome/background.js

```diff
@@ -368,27 +369,28 @@ const ThinBridgeTalkClient = {
     this.save();
   },
 
+  checkRedirectIntervalLimit(tabId, url) {
+    const intervalLimit = 1000;
+    let skip = false;
+    for (const key in this.recentRedirections) {
+      if (Math.abs(Date.now() - this.recentRedirections[key].time) > intervalLimit)
```

Comment:
It's not serious issue, how about substitute `Date.now()` to a variable?

```
    const now = Date.now();
    for (const key in this.recentRedirections) {
      if (Math.abs(now - this.recentRedirections[key].time) > intervalLimit)
      ...
```

Reply:
Fixed. Thanks!